### ISSUE TITLE
Install on Mavericks fails

### ISSUE THREAD

**Issue by erikkaplun** [gimbo/cabal-macosx#12]
Using GHC 7.10.2 on OS X 10.9.5 fails with this output:

```
Configuring cabal-macosx-0.1.0...
Building cabal-macosx-0.1.0...
Preprocessing library cabal-macosx-0.1.0...
[1 of 4] Compiling Distribution.MacOSX.DG ( Distribution/MacOSX/DG.hs, dist/dist-sandbox-722e9b63/build/Distribution/MacOSX/DG.o )
[2 of 4] Compiling Distribution.MacOSX.Common ( Distribution/MacOSX/Common.hs, dist/dist-sandbox-722e9b63/build/Distribution/MacOSX/Common.o )
[3 of 4] Compiling Distribution.MacOSX.Dependencies ( Distribution/MacOSX/Dependencies.hs, dist/dist-sandbox-722e9b63/build/Distribution/MacOSX/Dependencies.o )

Distribution/MacOSX/Dependencies.hs:67:1: Warning:
    The import of ‘System.IO’ is redundant
      except perhaps to import instances from ‘System.IO’
    To import instances alone, use: import System.IO()

Distribution/MacOSX/Dependencies.hs:153:28: Warning:
    A do-notation statement discarded a result of type ‘Char’
    Suppress this warning by saying ‘_ <- char '\n'’
    or by using the flag -fno-warn-unused-do-bind

Distribution/MacOSX/Dependencies.hs:158:23: Warning:
    A do-notation statement discarded a result of type ‘Char’
    Suppress this warning by saying ‘_ <- char '\t'’
    or by using the flag -fno-warn-unused-do-bind

Distribution/MacOSX/Dependencies.hs:160:23: Warning:
    A do-notation statement discarded a result of type ‘Char’
    Suppress this warning by saying ‘_ <- char '('’
    or by using the flag -fno-warn-unused-do-bind

Distribution/MacOSX/Dependencies.hs:161:23: Warning:
    A do-notation statement discarded a result of type ‘[Char]’
    Suppress this warning by saying
      ‘_ <- manyTill (noneOf ")") (char ')')’
    or by using the flag -fno-warn-unused-do-bind

Distribution/MacOSX/Dependencies.hs:209:6: Warning:
    A do-notation statement discarded a result of type ‘ProcessHandle’
    Suppress this warning by saying ‘_ <- runCommand cmd’
    or by using the flag -fno-warn-unused-do-bind
[4 of 4] Compiling Distribution.MacOSX ( Distribution/MacOSX.hs, dist/dist-sandbox-722e9b63/build/Distribution/MacOSX.o )

Distribution/MacOSX.hs:66:8: Not in scope: ‘catch’
cabal: Error: some packages failed to install:
cabal-macosx-0.1.0 failed during the building phase. The exception was:
ExitFailure 1
```

**Comment by danfran** [comment]
Same error for 'El Capitan'

**Comment by jdreaver** [comment]
I think this might be an incompatibility error with GHC 7.10.2. I am getting the same error on Linux (I am trying to build `reactive-banana-wx`, which depends on `cabal-macosx`).

**Comment by danfran** [comment]
I have downloaded cabal-macosx 0.1.0 from hackage. I have unpacked it and changed the file `Distribution.MacOSX.hs` and added the line:

`import System.IO.Error`

changed the error 'catch' line that returns the error to:

``catchIOError` \err -> putStrLn ("Warning...etc)`

and recompiled it successfully.

Then I moved back to `reactive-banana-wx` from the sandbox and I have added:

`cabal sandbox add-source /your_path/cabal-macosx-0.1.0/`

and finally it has compiled.
